CONTRIBUTING.md

```diff
@@ -45,11 +45,7 @@ automatically fix any issues. If the fixes can't be performed automatically,
 then you will need to manually address them before submitting your PR.
 
 ```shell
-nbqa black "$notebook"
-nbqa pyupgrade "$notebook"
-nbqa isort "$notebook"
-nbqa flake8 "$notebook" --extend-ignore=W391,E501,F821,E402,F404,W503,E203,E722,W293,W291
-python3 -m tensorflow_docs.tools.nbfmt --remove_outputs "$notebook"
+docker run -v ${PWD}:/setup/app gcr.io/python-docs-samples-tests/notebook_linter:latest <your_notebooks>
```

Comment:
You should use the public gcr.io image instead of this one, which has restricted access.